### PR TITLE
Add profile : Mariekie Coetzee

### PR DESCRIPTION
**Pull request by MariekieCoetzee**
Thank you for great initiative!

### CODE REVIEW

File: content/profiles/template-please-copy.md [hunk 1]
```diff
@@ -1,33 +1,24 @@
 ---
-title: "Your Name"
```
Comment (Layla-P) [inline content/profiles/template-please-copy.md:2]: Hi Marieke!
Thanks for your profile!
However, you have edited the template :-). Could you make a copy of this file and title it `mariekie-coetzee.md` and revert this one (`template-please-copy.md`) back?

Layla

**Review by Layla-P — CHANGES_REQUESTED** [review]

File: content/profiles/template-please-copy.md (hunk 1 above)
Comment (MariekieCoetzee) [reply content/profiles/template-please-copy.md]: Hello Layla!

Nooo thats soo embarrasing 🙈  really sorry about that!

Comment (Layla-P) [reply content/profiles/template-please-copy.md]: Oh don't worry!! Happens to the best of us 🤗🤗

**Review by Layla-P — APPROVED** [review]

File: content/profiles/template-please-copy.md (hunk 1 above)
Comment (Layla-P) [reply content/profiles/template-please-copy.md]: I've merged your changes - but you haven't added your profile and cover image. May have disappeared in the reversion. Just make another PR to add them :-)

Comment (MariekieCoetzee) [reply content/profiles/template-please-copy.md]: o dear!  thanks for pointing it out!

**Pull request merged**